en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies.md

```diff
@@ -0,0 +1,114 @@
+     For more information, see [Download and Initialize the apictl]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl).
+2.  Log in to the WSO2 API-M in the environment by following the instructions in [Login to an Environment]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#login-to-an-environment).
+3.  Run the corresponding apictl command below to get (list) Common API Policies in an environment.
+
+    1. Get API Common Policies in an environment.
+
+        -   **Command**
+            ``` bash
+            apictl get policies api -e <environment>
+            ```
+            ``` bash
+            apictl get policies api --environment <environment>
+            ```
+            ``` bash
+            apictl get policies api --environment <environment> --all 
+            ```
+
+            !!! info
+                **Flags:**  
+                
+                -   Required :  
+                    `--environment` or `-e` : Environment to be searched  
+                -   Optional :
+                    `--all`: All Common API Policies available in the environment
+                    `--limit` or `-l` : Maximum number of Common API Policies to return (Default 25)
+                    `--format` : pretty-print environments using templates
+
+            !!! example
+                ```bash
+                apictl get policies api -e dev 
+                ```
+                ```bash
+                apictl get policies api --environment production --limit 15 
+                ```    
+                ```bash
+                apictl get policies api --environment production
+                ```  
+                ```go
+                apictl get policies api --environment production --all 
+                ```  
+
+        -   **Response**
+
+            ```go
+            ID                                     NAME                       Display NAME               CATEGORY            APPLICABLE FLOWS     SUPPORTED GATEWAYS
+            e758ccd3-39ec-430c-8722-a44253a53ecd   ccCallInterceptorService   Call Interceptor Service   Mediation           [request response]   [ChoreoConnect]
+            55e8e47f-35ee-46b3-b097-5f347079c7c4   setToHeader                Set To Header              Mediation           [request]            [Synapse]
+            ```
+            
+            !!! tip 
+                When using the `apictl get policies api -e dev` command, `--all` optional flag can be used to 
+                get all available Common API Policies.
+
+            !!!note
+                Output of the `get policies api` command can be formatted with Go Templates. For more information on formatting the get commands, see [Formatting the outputs of get commands]({{base_path}}/install-and-setup/setup/api-controller/advanced-topics/formatting-the-output-of-get-command).
```

Comment:
Shall we mention that the --all flag and -l flag cannot be used at once?

Reply:
Fixed by [6465f5a](https://github.com/wso2/docs-apim/pull/6103/commits/6465f5a2f5f93d8ca2645700a95ab612cd0d25ed)